modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Sidebar/CompareRuns.tsx

```diff
@@ -20,44 +20,45 @@ const CompareRun = () => {
 	const navigate = useNavigate();
 
 	return (
-		<div style={{width: '380px'}}>
+		<div>
 			<ClayLayout.Row>
 				<ClayLayout.Col>
 					<label htmlFor="runA">Run A</label>
```

Comment:
wouldn't it be better to put it in the translation?

```suggestion
					<label htmlFor="runA"> i18n.sub('run-x', a) </label>
```

Reply:
I agree, the a should be string

```suggestion

					<label htmlFor="runA"> i18n.sub('run-x', 'a') </label>
```